medicines/doc-index-updater/src/main.rs

```diff
@@ -35,6 +38,10 @@ async fn main() -> Result<(), Box<dyn error::Error>> {
     let create_state = state.clone();
     let delete_state = state.clone();
 
+    let cors = warp::cors()
+        .allow_methods(vec![Method::POST])
+        .allow_any_origin();
```

Comment:
this ok?

Reply:
updated now to block for anything but `localhost:3000`, which should keep this endpoint secure until we choose to release PARs upload 👍